VARIABLES.md

```diff
@@ -57,4 +57,6 @@
 | ranger_audit_solr_url | Ranger Solr audit provider configuration. | string | `` | no |
 | ranger_policy_manager_url | Ranger admin URL to synchronize policies. | string | `` | no |
 | secondary_vpcs | List of VPCs to associate with Service Discovery namespace. | list | `<list>` | no |
+| s3_lifecycle_policy_transition_period | No of days for transition to a different storage class using lifecycle policy. | string  | `30` | no |
+| s3_storage_class | Destination s3 storage class for transition in the lifecycle policy. | string  | `INTELLIGENT_TIERING` | no |
```

Comment:
s3 -> S3
Is there an AWS document we can link to that lists all the valid values for this string? If not, can we list the valid values here then?